### ISSUE TITLE
SA1125 UseShorthandForNullableTypes incorrectly reported in nameof expression

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#637]
The following expression incorrectly results in SA1125:

```csharp
string name = nameof(Nullable<int>);
```
